_episodes/03-what-is-open-source.md

```diff
@@ -101,6 +101,8 @@ The requirement is that *if* you do distribute it, you do so in conformance with
 So you can make changes to a piece of software and you're not required to share the derived work with anyone else.
 And you can finesse license compatibility issues by letting the end user put everything together rather than you shipping the combined work.
 
+If you are concerned about the licensing of your dependencies, there are numerous paid and free automated tools to check for license compatibility.  For example, [Github Licensed](https://github.com/github/licensed).
```

Comment:
I'm not familiar with this tool.  I took a quick look and it raised a couple of concerns for me:
1) It seems to be specifically for Ruby.  If so, you should probably mention that in the text to avoid setting expectations too high.  But maybe better if there were a more general tool, if you know one.
2) I'm not clear how it makes decisions as to what licenses are compatible.  Do you simply put those you consider acceptable into the config file?  So the tool is not providing "advice" as to which licenses are considered compatible?

Reply:
Good catch, I didn't notice they dropped support for non-ruby code in version 4.  I'll remove that and add a warning that tools don't remove your liability and could generate false negatives.